### ISSUE TITLE
Add GitOps Application to manage this config

### ISSUE THREAD

**Issue by gnunn1** [gnunn-gitops/acm-hub-bootstrap#1]
Since I am bootstapping the ACM Hub and policies via this repo I am currently running the commands manually on a fresh Hub cluster. However one of the policies that is bootstrapped deploys OpenShift GitOps, to maintain the ACM configuration from this repo as changes are made we can create Application(s) that point back to this repo. This will eliminate having to manually update the Hub as things changes in this repo.

Basically the steps would be:

1. Provision Hub cluster
2. Bootstrap ACM with commands in this repo
3. GitOps is automatically bootstrapped by an ACM policy
4. In the cluster-config repo have Application(s) that point back to this repo to manage Hub and Policies
5. ACM Configuration and Policies are now kept in sync automatically by OpenShift GitOps
